### PR TITLE
Runtime dependencies paths compatible with NPMv3

### PR DESCRIPTION
**Pull request by yangbin**

### CODE REVIEW

**Comment by jieyanhuang** [comment]
should probably rebase your branch first :cold_sweat:

File: server.js [hunk 1]
```diff
@@ -4,14 +4,23 @@ var express = require('express'),
...
+app.use('/lib', express.static(resolve('jxml/public/lib/jxml.js')));
+app.use('/lib', express.static(resolve('systemjs/dist/system.js')));
+app.use('/lib', express.static(resolve('es6-module-loader/dist/es6-module-loader.js')));
+app.use('/lib', express.static(resolve('traceur/bin/traceur.js')));
 app.use('/lib/jx', express.static(__dirname + '/jx'));
-app.use('/lib/jx', express.static(__dirname + '/node_modules/jxml/jx'));
-app.use('/lib/jx/data', express.static(__dirname + '/node_modules/datanode'));
-app.use('/lib/datanode', express.static(__dirname + '/node_modules/datanode'));
+app.use('/lib/jx', express.static(resolve('jxml/jx/JXML')));
+app.use('/lib/jx', express.static(resolve('datanode')));
```
Comment (jieyanhuang) [inline server.js]: should this be `lib/jx/data`?

Comment (yangbin) [inline server.js]: actually, it should be just `/lib` since `resolve('datanode')` returns the directory with `jx/data`

**Comment by jieyanhuang** [comment]
I guess this looks fine :+1:

**Pull request merged**